src/backend/templates/calendar.twig

```diff
@@ -0,0 +1,171 @@
+	<style>	
+	page { size: A2 }
+
+  body.A2           .sheet { width: 420mm; height: 594mm }
+  body.A2.landscape .sheet { width: 595mm; height: 419mm }
+  /** Fix for Chrome issue #273306 **/
+  @media print {
+            body.A2.landscape { width: 595mm }
+    body.A2  body.A3.landscape { width: 420mm }
+  }
+
+	*, *:before, *:after {
+		box-sizing: border-box;
+		padding: 0;
+		margin: 0;
+	}
+	body {
+		font-family: 'Roboto';
+	}
+
+  a, a:visited {
+    color: red;
+    text-decoration: none;
+  }
+
+  section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .logo {
+    width: 150mm;
+  }
+
+  .header {
+    width: 100%;
+    height: 200mm;
+    font-size: 2em;
+
+    background: url('/static/hs-hackaton.jpg');
+    background-size:     cover;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: flex-start;
+  }
+
+  .header-box{
+    background: rgba(255,255,255,0.8);
+    padding: 10mm 20mm;
+    margin-bottom: 20mm;
+  }
+
+  .title {
+    text-align: center;
+    font-size: 2.1em;
+    letter-spacing: 0.15em;
+    margin: 15mm;
+    text-transform: uppercase;
+  }
+
+  .content {
+    flex-grow: 1;
+    font-size: {{ fontSize }}rem;
+  }
+
+  .content > div {
+    display: grid;
+    justify-content: center;
+
+    grid-gap: 0 1em;
+		grid-template-columns: repeat(2, 1fr);
+    grid-auto-flow: column dense;
+  }
+
+  .first-chunk {
+    grid-column-start: 1;
+    grid-column-end: 2;
+  }
+  .second-chunk {
+    grid-column-start: 2;
+    grid-column-end: 3;
+  }
+
+  .date-day {
+    color: red;
+    border-bottom:0.1em solid red;
+    font-size: 1.3em;
+    margin-top: 8mm;
+    margin-bottom: 3mm;
+  }
+
+  .event {
+    display: flex;
+    flex-direction: row;
+    text-align: left;
+    padding-left: 10mm;
+  }
+
+  .event .date {
+    font-weight: bold;
+    margin-right: 0.5em;
+  }
+
+  .footer {
+    font-size: 1.3em;
+    text-align: justify;
+  }
+	</style>
+</head>
+<body class="A2">
+	<section class="sheet padding-10mm">
+    <div class="header">
+      <div class="header-box">
+        <img class="logo" src="/static/logo.png" ></img>
+      </div>
+    </div>
+    <div class="title">
+      <h1>Kalendarz Wydarzeń</h1>
+    </div>
+    <div class="content">
+      <div>
+        {% for eventsInDay in eventsFirst %}
+          <div class="first-chunk">
+            <div class="date-day">
+              <b>{{ eventsInDay[0].date|date('d.m') }}</b>
+              {{ polishDayNames[eventsInDay[0].date.getDay()] }}
+            </div>
+            {% for event in eventsInDay %}
+              <div class="event">
+                <span class="date">{{ event.date|date("H:i") }}</span>
+                {{ event.title }}
+              </div>
+            {% endfor %}
+          </div>
+        {% endfor %}
+
+        {% for eventsInDay in eventsSecond %}
+          <div class="second-chunk">

```

Comment:
I think it would be just better to pass additional parameter in object instead of creating two variables here because a lot of duplicated code occurs and it can be dangerous.